### PR TITLE
Update getting started guide

### PR DESCRIPTION
**Pull request by sspinali**

### CODE REVIEW

File: content/en/docs/get-started/local.md [hunk 1]
```diff
@@ -19,13 +23,16 @@ Vitess supports MySQL 5.6+ and MariaDB 10.0+. We recommend MySQL 5.7 if your ins
 ```
 # Ubuntu based
 sudo apt install -y mysql-server etcd
+sudo apt install -y curl
```
Comment (morgo) [inline content/en/docs/get-started/local.md]: We should be able to merge these together as one line.

File: content/en/docs/get-started/local.md [hunk 2]
```diff
@@ -19,13 +23,16 @@ Vitess supports MySQL 5.6+ and MariaDB 10.0+. We recommend MySQL 5.7 if your ins
...
 
 # Debian
 sudo apt install -y default-mysql-server default-mysql-client etcd
+sudo apt install -y curl
 
 # Yum based
 sudo yum -y localinstall https://dev.mysql.com/get/mysql57-community-release-el7-9.noarch.rpm
 sudo yum -y install mysql-community-server etcd
+sudo yum -y install curl
 ```
```
Comment (morgo) [inline content/en/docs/get-started/local.md:34]: Same as above - these lines should merge.

File: content/en/docs/get-started/local.md [hunk 3]
```diff
@@ -60,7 +67,7 @@ sudo setenforce 0
 
 ## Configure Environment
 
-Add the following to your `.bashrc` file. Make sure to replace `/path/to/extracted-tarball` with the actual path to where you extracted the latest release file:
+Add the following to your `$Home/.bashrc` file. Make sure to replace `/path/to/extracted-tarball` with the actual path to where you extracted the latest release file:
```
Comment (morgo) [inline content/en/docs/get-started/local.md]: I believe this needs to capitalize to be correct. So `$HOME/.bashrc`

File: content/en/docs/get-started/local.md [hunk 4]
```diff
@@ -69,7 +76,7 @@ export VTDATAROOT=${HOME}/vtdataroot
 export PATH=${VTROOT}/bin:${PATH}
 ```
 
-You are now ready to start your first cluster!
+You are now ready to start your first cluster! Open a new terminal window before continuing.
```
Comment (morgo) [inline content/en/docs/get-started/local.md]: Can we instead say "open a new terminal window to ensure your `.bashrc` file changes take effect."

File: content/en/docs/get-started/local.md [hunk 5]
```diff
@@ -160,7 +168,7 @@ mysql> show tables;
 3 rows in set (0.01 sec)
 ```
 
-It is recommended to configure the MySQL command line to default to these settings, as the user guides omit `-h 127.0.0.1 -P 15306` for brevity:
+It is recommended to configure the MySQL command line to default to these settings, as the user guides omit `-h 127.0.0.1 -P 15306` for brevity. Enter the following into the command line:
```
Comment (morgo) [inline content/en/docs/get-started/local.md]: Can we say "Paste the following" instead.

File: content/en/docs/user-guides/vertical-split.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ weight: 6
 This guide follows on from [get started with a local deployment](../../get-started/local). It assumes that the `./101_initial_cluster.sh` script has been executed, and you have a running Vitess cluster.
 {{< /info >}}
 
-Vertical Split enables you to move a subset of tables to their own keyspace. Continuing on from the ecommerce example started in the get started guide, as your database continues to grow, you may decide to separate the `customer` and `corder` tables from the `product` table.  Let us add some data into our tables to illustrate how the vertical split works:
+Vertical Split enables you to move a subset of tables to their own keyspace. Continuing on from the ecommerce example started in the get started guide, as your database continues to grow, you may decide to separate the `customer` and `corder` tables from the `product` table.  Let us add some data into our tables to illustrate how the vertical split works. Enter the following into the command line:
```
Comment (morgo) [inline content/en/docs/user-guides/vertical-split.md]: I prefer "Paste the following.." again here

**Review by morgo — COMMENTED** [review]
Thank you for helping improve the getting started guides! Just a quick note so you know some changes coming down the pipeline...

I have a PR open in https://github.com/vitessio/vitess/pull/5654 which eliminates the need to set VTROOT. We can make a further change after it merges.

**Comment by morgo** [comment]
LGTM. It's still marked as a DRAFT, is it okay to merge?

**Comment by sspinali** [comment]
It's all ready to merge.
    On Tuesday, January 7, 2020, 01:09:06 PM PST, Morgan Tocker <notifications@github.com> wrote:  
 
 
LGTM. It's still marked as a DRAFT, is it okay to merge?

**Review by morgo — APPROVED** [review]
LGTM Thank you for helping improve the docs!

**Pull request merged**